packages/nuejs.org/docs/css-best-practices.md

```diff
@@ -118,7 +118,7 @@ article {
 }
 ```
 
-Simple selectors make easy-to-read and easy-to-maintain CSS, keep your file sizes to a minimum, and use lower specificity for overrides.
+Simple selectors are easy-to-read and easy-to-maintain. Keep your file sizes to a minimum and use lower specificity for overrides.
```

Comment:
Your sentence reads better, but changes the meaning. ie. good CSS --> good selectors

Reply:
Yes you are right. What about this new change:
Simple selectors make your CSS easier to read and easier to maintain. They keep your file size to a minimum and use lover specificity for overrides.
You can also use easy in the sentence. Your choice.

Reply:
```suggestion
Simple selectors make your CSS easy to read and maintain. They keep your file sizes to a minimum and make you use lower specificity for overrides.
```
?

Reply:
another way of saying it would be:
They keep your file size to a minimum and allow you to use lower specificity for overrides.

Reply:
@tipiirai 

Do you agree with [this suggestion] (#discussion_r1705705790) or would you like some changes?

You can just edit my comment if you want to do some changes and then commit the suggestion if you like it.

Reply:
How about this below. If you don't link it then I am very happy with your suggestion.
They help keep your file size small and allow you to use lower specificity when making overrides.

Reply:
Edited the code piece again.